docs/cloud/workspaces.md

```diff
@@ -45,22 +45,33 @@ Click **Add**.  The user will receive an email invitation to join the organizati
 
 To remove a user from the organization, select the options menu button (hamburger) to the right of the user and click **Remove**.
 
-## Rebooting and Disabling Workspaces
+## Workspace Sleeping
 
-Issues with workspaces are uncommon, but if you ever need to reboot your workspace, you can do so from the **Settings** > **Advanced** tab.
-
-Under the **Workspace Status** section choose the `Reboot` option. This will cycle the workspace, which will cause a brief interruption in service. This typically takes no longer than 30 seconds.
+Steampipe Cloud will put workspaces to sleep if they have been inactive for 7 days.  When a workspace is sleeping the database is effectively shut down.  You will not be able to query the workspace via any means until you wake it again, and all pipelines will be disabled.
 
-You'll also find a **Disable** option there, which will effectively shut down the workspace. This means that all connections to the workspace will be terminated, and you will not be able to query the workspace via any means until you `Enable` it again. All pipelines will also be disabled as part of this change.
+Any sleeping workspaces will show a persistent banner at the top of the workspace page as a reminder that the workspace is sleeping.  
 
-<img src="/images/docs/cloud/cloud-workspace-state-management.png" width="400pt"/>
+<img src="/images/docs/cloud/cloud-workspace-disabled-banner.png" width="400pt"/>
 <br />
 
-Any disabled workspaces will show a persistent banner at the top of the workspace page as a reminder that the workspace is not enabled.
+To wake up your workspace, click the **Wake Up** button on the banner.  Your workspace will be started and available within a few seconds.
```

Comment:
Think `a few seconds` may make people think like 3 or 4? It could be 30+.

```suggestion
To wake up your workspace, click the **Wake Up** button on the banner.  Your workspace will be started and available within a short period.
```